scripts/release.sh

```diff
@@ -5,9 +5,9 @@ if [[ -z $(git status -s) ]]
 then
   if [[ "$VIRTUAL_ENV" != ""  ]]
   then
-    pip install --upgrade setuptools wheel twine
+    pip install --upgrade setuptools wheel twine build
     rm -rf dist/ build/
-    python setup.py sdist bdist_wheel
+    python -m build
```

Comment:
I diffed the produced `tar.gz` and `whl` packages and they don't include any unexpected changes.